models/person.go

```diff
@@ -48,21 +44,17 @@ func (c Person) Subject() string {
 }
 
 // SetOrganizationInfo allow organization relationships to be passed in
```

Comment:
This comment is no longer correct.  Can you run golint on your changes before you push?  I'm using go-plus in Atom which has this integrated into my editor, which makes it easier to see these issues.

Reply:
Thanks. Just installed go-plus.